Test/browser setup

PixnBits opened the pull request:
I noticed there were no automated tests. I'd wager it is probably effective to start with browser testing than unit tests in the event of some implementation refactoring. This is a setup with Playwright, with the start of some tests. I can continue to expand the tests if this approach is desirable, or change course based on maintainer feedback :+1:

PixnBits committed:
test(browser): add playwright defaults

PixnBits committed:
test(browsers): add devices CTA

bmv437 commented on package.json:
```diff
@@ -7,7 +7,9 @@
-    "check": "biome check .",
+    "test": "npm run test:static && npm run test:browsers",
+    "test:static": "biome check .",
+    "test:browsers": "playwright test",
```
Nitpick: The original `check` script is a format/lint checker, and not really a kind of test. Tests would commonly be something like `test:e2e` (playwright) and `test:unit` (vitest+testing-library).

bmv437 replied:
@PixnBits Let me know if this sounds like a reasonable change. I'd love to see this PR get merged, and happy to assist if needed.

PixnBits replied:
sorry for the delay
I'd argue that static code analysis is testing too ;)
but I'll update this to separate out tests from other checks :+1:

PixnBits replied:
updated in 3c9088b

PixnBits committed:
Merge branch 'master' of github.com:meshtastic/web into test/browser-setup

PixnBits committed:
style(test): format example Playwright tests to project's

PixnBits committed:
refactor(test): disable any check in sample test

the test file is an example, does not test this project in any way, and will be removed as tests for this project are written and the example is no longer needed

PixnBits commented on tests/example.spec.ts:
I'll remove this example file before moving the PR to ready for review :saluting_face:
